src/lib/time/__init__.js

```diff
@@ -40,10 +135,16 @@ var $builtinmodule = function (name) {
         return susp;
     });
 
+    mod.sleep.__doc__ = 
+        "sleep(seconds)\n" +
+        "\n" +
+        "Delay execution for a given number of seconds.  The argument may be\n" +
+        "a floating point number for subsecond precision.";
+
     function isLeapYear(year) {
         if((year & 3) != 0) return false;
         return ((year % 100) != 0 || (year % 400) == 0);
-    };
+    }
 
     function getDayOfYear(date) {
```

Comment:
I think it is a quick fix, as I've already tested it.  in UTC+ and UTC- timezones and it passes all the tests in both locations.